setup.py

```diff
@@ -9,6 +10,21 @@
     changes = f.read()
 
 
+install_requires = [
+    'pytest',
+    'pep8',
+    'pyflakes',
+    'coverage',
+    'sphinx',
+    'pytest-cov',
```

Comment:
You're right that they shouldn't be in install_requires. I moved them all because it felt more consistent but see below re. test_requires etc.